### DISCUSSION TITLE
How can I use a custom query builder like laravel-nestedset package query to get the Category and children as a tree in the table.

### DISCUSSION THREAD

**Discussion by DevYSM** [filamentphp/filament#10865] [Help]
Hello,
How can I implement the [https://github.com/lazychaser/laravel-nestedset](url) instead of ORM builder?

**Comment by zipavlin** [comment]
@DevYSM did you perhaps manage to figure this out?

**Reply by DevYSM** [reply]
I have forked this package and overwite it. 
https://filamentphp.com/plugins/solution-forest-tree

**Reply by zipavlin** [reply]
As a private repo I guess? I know this is a huge ask, but could I maybe take a look at your fork? :)
I also decided for this plugin. It's close to what we would need, but not quite there. I am/was thinking about rewriting filement's Table to support nesting, but it seems quite a lot of work and it wouldn't be optimal for future Filament releases.

**Reply by DevYSM** [reply]
Sorry, I can't share it with you because it's my company's work, and according to the company rules, don't share company code with anyone. 
But if you'd like to make it into the table, you can use the custom view and HTML table to do that. 
If you need any help, can you connect with me to help you.

**Reply by zipavlin** [reply]
@DevYSM sorry for not responding for so long. I completely understand, thank you anyway!
I actually figured that it's pretty straight forward if I don't try to implement drag and drop, so I just wrote Actions for default table and it works great.
